shared/AppInsightsCore/Tests/Selenium/CorePerfCheck.Tests.ts

```diff
@@ -159,15 +159,15 @@ export class CorePerfCheckTests extends AITestClass {
                     objForEachKey(baseTestObject, (name, value) => {
                         checks++;
                     });
-                }, 300, iterations, 0.003).then((baseDuration) => {
+                }, 350, iterations, 0.003).then((baseDuration) => {
```

Comment:
Seen several failures on build servers where the total time was between 300-310ms so added some additional space, but leaving the average check @ 0.003ms